### ISSUE TITLE
Volume Visualize ql: No enteries in scalar or vector dropdown

### ISSUE THREAD

**Issue by bonsak** [qLab/qLib#468]
Shouldn't the "scalarfield" and the "vectorfield" right-hand dropdowns be populated with the available volumes in the stream the node is connected to? Tried with both Houdini volumes and VDB's. Get "no enteries" with both.

**Comment by johnnyquest** [comment]
Yeah, that's a totally experimental node, so you actually have to delete all but the single volume you want to visualize (or xyz volumes in case of velocity).

**Comment by bonsak** [comment]
Thanks.

**Comment by johnnyquest** [comment]
You don't need to close the issue -- since it's experimental, it needs development, so there's nothing wrong keeping the ticket open.
